scripts/main.py

```diff
@@ -345,18 +348,64 @@ def main():
         np.random.seed(args.seed)
         torch.manual_seed(args.seed)
         torch.cuda.manual_seed(args.seed)
+
     # train & resume
     ##########################################################################
     if args.mode == "train":
-        args.files_train = gather_files_from_cmdline(args.files_train)
-        args.val_files = gather_files_from_cmdline(args.val_files)
-        _logger.debug("Training data:   " + "\n".join(args.files_train))
+
+        args.cleanpeakfile = gather_files_from_cmdline(args.cleanpeakfile)
```

Comment:
would be good to specify the extension for `cleanpeakfile` just to be explicit about the type

Reply:
It takes in both narrowPeak and BED types. Found it hard to encode that into the name.

Reply:
can't it be with `extension=("ex1", "ext2")`?